INFO  | 12/06/2019 06:28:18 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN/GRANMixtureBernoulli_grid_2019-Dec-06-18-28-18_20764/log_exp_20764.txt
INFO  | 12/06/2019 06:28:18 PM | File run_exp.py           | Line 27    | Exp instance id = 20764
INFO  | 12/06/2019 06:28:18 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 12/06/2019 06:28:18 PM | File run_exp.py           | Line 29    | Config =
ERROR | 12/06/2019 06:28:18 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "run_exp.py", line 36, in main
    runner = eval(config.runner)(config)
  File "/Users/ndemir/Developer/CS224w/graph-generation/GRAN/runner/gran_runner.py", line 105, in __init__
    assert self.use_gpu == True
AssertionError

